src/vs/workbench/services/extensions/common/extensionsApiProposals.ts

```diff
@@ -127,6 +127,7 @@ export const allApiProposals = Object.freeze({
 	treeViewReveal: 'https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.treeViewReveal.d.ts',
 	tunnelFactory: 'https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.tunnelFactory.d.ts',
 	tunnels: 'https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.tunnels.d.ts',
+	viewContainerContext: 'https://raw.githubusercontent.com/microsoft/vscode/main/src/vscode-dts/vscode.proposed.viewContainerContext.d.ts',
```

Comment:
By our convention this should be called `contribXYZ`

Reply:
Updated reference (in new format) and filename to follow this convention